draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "RTP over QUIC (RoQ)"
+title: "RTP and RTCP over QUIC (RoQ)"
```

Comment:
I agree with the other changes, but I don't think it is necessary to change the title. RTCP is part of RTP in RFC 3550, which only has RTP in its title.

Reply:
> I agree with the other changes, but I don't think it is necessary to change the title. RTCP is part of RTP in RFC 3550, which only has RTP in its title.

@mengelbart - thank you for pushing back on this change. My intention going into this PR was to make it obvious that what we said about encapsulation applied to both RTP Data Transfer Protocol and RTP Control Protocol, but after I added the Note to the Reader about this, I ought to have reversed the proposed change to the title. 

And I am slightly cursing under my breath that "RTCP" caught on, but "RTDTP" never did! :face_with_spiral_eyes: